### ISSUE TITLE
Error building production

### ISSUE THREAD

**Issue by lorentzyip** [HaithemMosbahi/ngx-avatar#36]
Hi, I've got error when building production app (`ng build --prod`) with ngx-avatar 3.0.0. The error message is:

ERROR in : Unexpected value 'AvatarModule in C:/path/node_modules/ngx-avatar/ngx-avatar.d.ts' imported by the module 'XXX in C:/path/XXX/XXX.module.ts'. Please add a @NgModule annotation.

Environment:
@angular/core@6.1.7
@angular/cli@6.2.2
ngx-avatar@3.0.0

Please help.

**Comment by lorentzyip** [comment]
Let me give more information, the XXX module is a lazy-loading module which imports the AvatarModule. Here are the source codes:

app.module.ts
```
@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    BrowserModule,
    AppRoutingModule
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```
app-routing.module.ts
```
const routes: Routes = [
  {
    path: '',
    redirectTo: 'xxx',
    pathMatch: 'full'
  },
  {
    path: 'xxx',
    loadChildren: './xxx/xxx.module#XxxModule'
  }
];

@NgModule({
  imports: [RouterModule.forRoot(routes)],
  exports: [RouterModule]
})
export class AppRoutingModule { }
```
xxx.module.ts
```
@NgModule({
  imports: [
    CommonModule,
    AvatarModule,
    XxxRoutingModule
  ],
  declarations: [XxxComponent]
})
export class XxxModule { }
```
xxx-routing.module.ts
```
const routes: Routes = [
  {
    path: '',
    component: XxxComponent
  }
];

@NgModule({
  imports: [RouterModule.forChild(routes)],
  exports: [RouterModule]
})
export class XxxRoutingModule { }
```
xxx.component.ts inside xxx module
```
@Component({
  selector: 'app-xxx',
  templateUrl: './xxx.component.html',
  styleUrls: ['./xxx.component.css']
})
export class XxxComponent implements OnInit {
  constructor() { }

  ngOnInit() {
  }
}
```
xxx.component.html 
```
<ngx-avatar value="HW"></ngx-avatar>
```

**Comment by HaithemMosbahi** [comment]
@lorentzyip Thank you for raising this issue. I managed to reproduce it and yes it's a bug.

The thing is I just migrated the component to use Angular 6 and Angular CLI for building and packaging and it seems that there's an issue with AOT mode because when I run this `ng build --prod --aot=false --build-optimizer=false` it works.

I am working on this bug and I will do my best to fix it as soon as possible,

**Comment by HaithemMosbahi** [comment]
Hi @lorentzyip I just released a new version 3.0.1 that contains a partial fix for the build prod issue. 
Importing AvatarModule should work now, please let me now if it works for you. 

For now the only thing that doesn't work with prod mode is AvatarModule.forRoot(avatarConfig). I'm working on it to make it AOT friendly

**Comment by lorentzyip** [comment]
@HaithemMosbahi Thank you so much. 👏👏👏
